### DISCUSSION TITLE
Is it possible to default initial position and size of player?

### DISCUSSION THREAD

**Discussion by AtomicNess123** [kljohann/mpv.el#28] [Q&A] [answered]
Basically that. Would be nice to set this up to have in my case video on a corner, so I can take notes.

**Comment by kljohann** [comment, marked as answer]
You can use the [`--geometry` option](https://mpv.io/manual/master/#options-geometry). Whether this works might depend on your window manager, though.
```elisp
(setq mpv-default-options '("--geometry=100%:100%"))
```

**Reply by AtomicNess123** [reply]
Thanks! Is there also an option to make the window be on top above any other windows?

**Reply by kljohann** [reply]
Maybe [--focus-on-open](https://mpv.io/manual/master/#options-focus-on-open) or [--ontop](https://mpv.io/manual/master/#options-ontop) does what you want?

**Reply by AtomicNess123** [reply]
Thanks, but how to I concatenate two options together in elisp? 

Like this?: `(setq mpv-default-options '("--geometry=100%:100%" " "--focus-on-open"))`

**Reply by kljohann** [reply]
```elisp
(setq mpv-default-options '("--geometry=100%:100%" "--focus-on-open"))
```
should work

**Reply by AtomicNess123** [reply]
Thanks for much. By the way, the option that works for me is `--ontop `(not `--focus-on-open`).
